README.md

```diff
@@ -1,2 +1,207 @@
 # CommandLine
 C++ lib for command line parsing 
+
+ifm3d
+=====
+Library and utilities for working with ifm pmd-based 3D ToF Cameras.
```

Comment:
why ifm3d readme in this commit

Reply:
removed., Sorry it got added accidentally.